### DISCUSSION TITLE
Proper way to update apps (nginx as an example)

### DISCUSSION THREAD

**Discussion by yuriw** [tteck/Proxmox#2447] [General]
Hello all

I tried updating nginx LCX by running the script used for the initial setup from the LCX shell.  It failed at the end and messed up my ngnix completely.

Not a big loss as I had a backup :)

However, how do other people update the apps/services installed by the @tteck wonderful scripts?

TIA

**Comment by tteck** [comment]
The intended way is to type the command `update` in the LXC console.

**Reply by yuriw** [reply]
... and it worked, thx!

I thought that rerunning the install script would be the same, but it is not
